### PR TITLE
Download the JetBrainsMono font directly from GitHub

### LINKED ISSUE #7: Consider changing the download location and version of the JetBrains Mono font
# 🚀 Feature Proposal #

1. This Ansible role uses [the font "JetBrains Mono"](https://www.jetbrains.com/lp/mono/), downloading it from `jetbrains.com`.  Since this font is [open source on GitHub](https://github.com/JetBrains/JetBrainsMono), do we want to download the font assets from there instead? 
2. Note that the font is currently quite a few versions ahead of what we are downloading in this Ansible role.  Should we be using the newer version?

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request changes teh Ansible code to download [the JetBrainsMono font](https://www.jetbrains.com/lp/mono/) directly from [its GitHub repository](https://github.com/JetBrains/JetBrainsMono).

Also download a newer version, as identified in the variable `jetbrainsmono_version` in the new file `defaults/main.yml`, and update the `README.md` file accordingly.

## 💭 Motivation and context ##

Resolves #7.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Seems like a win to me. 🏆

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ None.
 
 ## Role Variables ##
 
-None.
+- `install_directory` - the version of the JetBrainsMono font to
```
Comment (mcdonnnj) [inline README.md:36]: I think this was missed.

```suggestion
- `jetbrainsmono_version` - the version of the JetBrainsMono font to
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Nice improvement! Just one change and then it LGTM.

**Review by mcdonnnj — APPROVED** [review]
:shipit: 🚀🌕

**Pull request merged**